hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/shell/TestOzoneRepairShell.java

```diff
@@ -160,4 +162,21 @@ public void testQuotaRepair() throws Exception {
       return false;
     }, 1000, 10000);
   }
+
+  @Test
+  public void testFSORepair() throws Exception {
+    CommandLine cmd = new CommandLine(new OzoneRepair()).addSubcommand(new CommandLine(new OMRepair())
+            .addSubcommand(new FSORepairCLI()));
```

Comment:
Why do we have this separate test case?  Isn't it redundant with one of the cases in `TestFSORepairTool`?

```suggestion
    CommandLine cmd = new OzoneRepair().getCmd();
```